unity-renderer/Assets/Scripts/MainScripts/DCL/Components/AvatarAttach/AvatarAttachComponent.cs

```diff
@@ -56,7 +56,7 @@ IEnumerator IComponent.ApplyChanges(BaseModel newModel)
 
         void IComponent.RaiseOnAppliedChanges() { }
 
-        Transform IMonoBehaviour.GetTransform() => handler.entity?.gameObject.transform;
+        Transform IMonoBehaviour.GetTransform() => null;
```

Comment:
Is this change expected?

Reply:
yes, `IMonoBehaviour.GetTransform` is only used for "pointer events" components so is not necessary to set something there

Reply:
Also, this component does not create any `GameObject` so it makes no sense setting something there :P